serverjs/cubefn.js

```diff
@@ -315,15 +316,13 @@ const methods = {
     if (!seed) {
       seed = Date.now().toString();
     }
-
-    const pack = util
-      .shuffle(cube.cards, seed)
-      .slice(0, 15)
-      .map((card) => carddb.getCardDetails(card));
-
+    cube.cards = cube.cards.map((card) => ({ ...card, details: { ...carddb.getCardDetails(card) } }));
```

Comment:
we don't know if cube has been called with `.lean()`, let's not populate the details here

Reply:
We have to populate details before calling `createDraft` in case it is a custom draft format where filters refer to card details. We might need to populate elo and price as well, but I think that since those are dynamic values not supporting them in this case for custom draft formats is likely okay.